features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/timeline/components/MessagesReactionButton.kt

```diff
@@ -150,13 +156,24 @@ private fun ReactionContent(
     verticalAlignment = Alignment.CenterVertically,
     modifier = modifier,
 ) {
-    Text(
-        text = reaction.displayKey,
-        style = ElementTheme.typography.fontBodyMdRegular.copy(
-            fontSize = 15.sp,
-            lineHeight = reactionEmojiLineHeight,
-        ),
-    )
+    if (reaction.key.startsWith("mxc://")) {
+        BlurHashAsyncImage(
```

Comment:
Same about this one: if it doesn't have a blurhash, `AsyncImage` should be a better option